lms/djangoapps/gating/tests/test_api.py

```diff
@@ -111,3 +111,11 @@ def test_no_gated_content(self, mock_score):
 
         evaluate_prerequisite(self.course, self.subsection_grade, self.user)
         self.assertFalse(mock_score.called)
+
+    @data((59.50, 100), (60.50, 100))
+    @unpack
+    def test_subsection_percentage_rounding(self, mock_earned, mock_possible):
+        self.subsection_grade = Mock(graded_total=Mock(earned=mock_earned, possible=mock_possible))
```

Comment:
do you want to update `self.subsection_grade` or you can use the local var here?